### DISCUSSION TITLE
Why are my args table not loading properly in React?

### DISCUSSION THREAD

**Discussion by JamDev0** [storybookjs/storybook#20680] [Help]
Currently, my argument table looks like this:

![Captura de tela_20230118_190934](https://user-images.githubusercontent.com/61752887/213306665-abec21ae-364e-43f6-8aec-0777499a7700.png)

In those spaces, the argument type (string, number, etc.) should be displayed:

![Captura de tela_20230118_190934](https://user-images.githubusercontent.com/61752887/213306820-83c1e6ca-611e-4772-9c80-205348ed5934.png)

And in these, the default values that I have set should be displayed.:

![Captura de tela_20230118_190934](https://user-images.githubusercontent.com/61752887/213307128-616f01ea-a005-4942-95e0-c58843defbc0.png)

Does anyone know why it is like that?

**Comment by chakAs3** [comment]
what framework are you using?  react , vue ?

**Reply by JamDev0** [reply]
React
